.github/workflows/nm-build-test.yml

```diff
@@ -173,7 +173,7 @@ jobs:
 
     # update docker
     DOCKER:
-        needs: [BUILD]
+        needs: [BUILD, UPLOAD]
```

Comment:
nit: this can just be `UPLOAD`

Reply:
Yeah I think it'll be neat too just to have UPLOAD here, but one of the input arguments needs the whl value from the BUILD output so I have to add the BUILD here.